frontend/src/screens/apps/AppCreated.tsx

```diff
@@ -15,6 +15,24 @@ export default function AppCreated() {
   const [isPopupVisible, setPopupVisible] = useState(false);
   const popupRef = useRef<HTMLDivElement>(null);
 
+  useEffect(() => {
+    // dispatch a success event which can be listened to by the opener or by the app that embedded the webview
+    // this gives those apps the chance to know the user has enabled the connection
+    const nwcEvent = new CustomEvent("nwc:success", { detail: {} });
+    window.dispatchEvent(nwcEvent);
+
+    // notify the opener of the successful connection
+    if (window.opener) {
+      window.opener.postMessage(
+        {
+          type: "nwc:success",
+          payload: { success: true },
```

Comment:
@bumi do you know why the payload of this event sent to the window opener is different from the normal nwc:success event? shouldn't they be the same?